### ISSUE TITLE
Could not find module `@ember/component/helper`

### ISSUE THREAD

**Issue by cmackenz** [buschtoens/ember-route-task-helper#16]
Just noticed this issue with a fresh build of my project. When clicking on my `task` button I get the following error in my browsers console.

```sh
Error: Could not find module `@ember/component/helper` imported from `ember-route-task-helper/helpers/route-task`
    at missingModule (loader.js:247)
    at findModule (loader.js:258)
    at Module.findDeps (loader.js:168)
    at findModule (loader.js:262)
    at Module.findDeps (loader.js:168)
    at findModule (loader.js:262)
    at requireModule (loader.js:24)
    at Class._extractDefaultExport (resolver.js:382)
    at Class.resolveOther (resolver.js:79)
    at Class.superWrapper [as resolveOther] (ember.debug.js:40887)
```

`"ember-route-task-helper": "^0.3.0",`

```sh
ember-cli: 2.12.1
node: 6.11.4
os: darwin x64
```

**Comment by buschtoens** [comment]
Thanks for reporting. What version of ember-cli-babel are you using?

**Comment by cmackenz** [comment]
`"ember-cli-babel": "^5.1.7",`

**Comment by buschtoens** [comment]
Ah yes. I fucked up. 

Last paragraph here explains it. 
https://github.com/ember-cli/babel-plugin-ember-modules-api-polyfill/blob/master/README.md

A PR would be appreciated. Otherwise I'll get to fix it on Monday.

**Comment by buschtoens** [comment]
Fixed via https://github.com/buschtoens/ember-route-task-helper/pull/18 and released as `0.3.1`. :tada:
